open-ce/utils.py

```diff
@@ -216,3 +229,49 @@ def generalize_version(package):
                 package = name + operator + version + ".*" + build
 
     return package
+
+def cuda_level_supported(cuda_level):
+    '''
+    Check if the requested cuda level is supported by loaded NVIDIA driver
+    '''
+
+    if float(get_driver_cuda_level()) >= float(cuda_level):
+       return True
+    else:
+       return False
+
+def get_driver_cuda_level():
```

Comment:
This doesn't seem to be used anywhere.

Reply:
It's called a few lines earlier in cuda_level_supported